src/org/folio/eureka/EurekaImage.groovy

```diff
@@ -92,17 +92,46 @@ class EurekaImage implements Serializable {
     }
   }
 
+  def updatePL() {
```

Comment:
Could you put here complete function name in camel case?
It is not really clear how to expand it.

Reply:
Simple: PL == platform, how do you plan to use camel case to word platform?

Reply:
As to me **updateEurekaPlaform** is more self explaining function name.
In couple month later we will have poor code readability and have to guess what this function is.